docs/plugin-nabu.md

```diff
@@ -6,6 +6,15 @@ The Nabu plugin provides two input formats:
 * `nabu_file_qc` contains file QC information
 * `case_archive` contains information about a case archive's archiving status
 
+## Write Nabu Archive Case actions
+A Nabu Case Archive requires five inputs:
+* `caseIdentifier` (String)
+* `limsIds` (Set)
+* `requisitionId` (Int)
+* `workflowRunIdsForOffsiteArchive` (Set)
+* `workflowRunIdsForVidarrArchival` (Set)
+These values make up the archive instance which is stored in Nabu.
```

Comment:
Generally don't need to document functions in readmes since the Definitions page on shesmu handles that, but if you are going to, please elaborate more on what each of these represents, especially the distinction between 'OffsiteArchive' and 'VidarrArchival'